### PR TITLE
[17.0] [MIG] product_print_category

### PR DESCRIPTION
**Pull request by MarioLM-23**
Standar migration with changes in views, readme images and improve test

### CODE REVIEW

File: product_print_category/views/view_product_print_category.xml [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+                            </div>
+                        </button>
+
+                        <button
+                            class="oe_stat_button"
+                            name="action_view_product_product"
+                            type="object"
+                            icon="fa-list"
+                            context="{'to_print': False}"
+                            invisibile="product_qty == 0"
```
Comment (legalsylvain) [inline product_print_category/views/view_product_print_category.xml]:
```suggestion
                            invisible="product_qty == 0"
```

File: product_print_category/views/view_product_print_category.xml [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+    <record model="ir.ui.view" id="view_product_print_category_form_2">
+        <field name="model">product.print.category</field>
+        <field name="arch" type="xml">
+            <form>
+                <header>
+                    <button
+                        name="%(product_print_category.action_product_product_2_product_print_wizard)d"
+                        type="action"
+                        invisibile="product_to_print_qty == 0"
```
Comment (legalsylvain) [inline product_print_category/views/view_product_print_category.xml]:
```suggestion
                        invisible="product_to_print_qty == 0"
```

File: product_print_category/views/view_product_print_category.xml [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+                        string="Print Obsolete Products"
+                        class="oe_highlight"
+                        icon="fa-print"
+                    />
+                    <button
+                        name="%(product_print_category.action_product_product_2_product_print_wizard)d"
+                        type="action"
+                        invisibile="product_qty == 0"
```
Comment (legalsylvain) [inline product_print_category/views/view_product_print_category.xml]:
```suggestion
                        invisible="product_qty == 0"
```

**Review by legalsylvain — CHANGES_REQUESTED** [review]
Hi. Thanks for porting this module @mariobinhex.

Suggestion inline. Otherwise, LGTM. Thanks !

**Comment by MarioLM-23** [comment]
@legalsylvain Done! Sorry for that mistake

**Review by legalsylvain — APPROVED** [review]
> @legalsylvain Done! Sorry for that mistake

No problem.

**Review by dreispt — APPROVED** [review]

**Pull request merged**